web/adventures/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+def parse_URL_list_field(url_list, separator):
+    """Takes a string of urls and coverts them into a list."""
+    if url_list == "":
+        return []
+    return url_list.split(",")
+
+class URLListField(models.URLField):
```

Comment:
This may make more sense to say `class URLListField(models.Field):` instead. Thoughts?